swarm-test-cluster/main.tf

```diff
@@ -105,6 +105,12 @@ resource "docker_container" "swarm" {
     "--pprof",
     "--pprofaddr=0.0.0.0",
     "--ws",
+    "--tracing",
+    "--tracing.endpoint=172.18.0.1:6831",
```

Comment:
This is really brittle, assumes we have started `jaeger` on the `swarm` network.